src/coreclr/jit/hwintrinsiclistarm64sve.h

```diff
@@ -17,6 +17,10 @@
 //  SVE Intrinsics
 
 // Sve
+HARDWARE_INTRINSIC(Sve,           Abs,                                                              -1,     -1,      false, {INS_sve_abs,        INS_invalid,        INS_sve_abs,        INS_invalid,        INS_sve_abs,        INS_invalid,        INS_sve_abs,        INS_invalid,        INS_sve_fabs,       INS_sve_fabs},    HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_EmbeddedMaskedOperation)
+
+HARDWARE_INTRINSIC(Sve,           Add,                                                              -1,     -1,      false, {INS_sve_add,        INS_sve_add,        INS_sve_add,        INS_sve_add,        INS_sve_add,        INS_sve_add,        INS_sve_add,        INS_sve_add,        INS_sve_fadd,       INS_sve_fadd},    HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_EmbeddedMaskedOperation|HW_Flag_HasRMWSemantics|HW_Flag_LowMaskedOperation)
```

Comment:
Can you elaborate a bit on what `LowMaskedOperation` means?

I'm looking at the architecture manual and don't see any limitations on the `Add (predicated) instruction. The attached comment to the enum entry is `// The intrinsic uses a mask in arg1 to select elements present in the result, and must use a low register.`

Reply:
> Can you elaborate a bit on what `LowMaskedOperation` means?

The instruction only has 3bits for a predicate register so is limited to using predicate registers 0 to 7. This is quite a common pattern across Sve, hence using an common enum for it (I wouldn't do similar for the handful have only 2 bits)